src/ucp/rma/basic_rma.c

```diff
@@ -17,6 +17,10 @@
 #include <ucs/datastruct/mpool.inl>
 #include <ucp/core/ucp_ep.inl>
 
+typedef ucs_status_t 
+(*ucp_rma_send_func_t)(ucp_request_t *req, uct_rkey_t uct_rkey, 
+                       ucp_ep_rma_config_t *rma_config);
```

Comment:
could be 'const ucp_ep_rma_config_t *rma_config' to indicate this is not changed